doc/source/matcookbook.rst

```diff
@@ -446,7 +446,7 @@ aluminium atoms are actually randomly switched with chromium atoms.
 In general the NCrystal cfg-string syntax for defining a multi-phase material is
 ``phases<FRAC1*CFG1&..&FRACN*CFGN>[;COMMONCFG]``. Here, ``FRAC1`` is the
 fraction of phase 1, which is defined by the NCrystal cfg-string ``CFG1``, and
-so forth. The indicated fractions are assumed to by volume fractions, and must
+so forth. The indicated fractions are assumed to be volume fractions, and must
```

Comment:
Actually "by volume" is intended here, but it might be better to write `be "by-volume"` ?